rocks.thorium.Thorium.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>rocks.thorium.Thorium</id>
+  <launchable type="desktop-id">rocks.thorium.Thorium.desktop</launchable>
+  <name>Thorium Browser</name>
+  <summary>A fast and optimized web browser based on Chromium</summary>
+  <description>
+    <p>Thorium Browser is a fast, privacy-focused web browser based on Chromium, optimized for performance and stability. Designed for power users and developers, it offers enhanced speed, lower memory usage, and more control over browser behavior. Thorium removes unnecessary Google tracking, giving users a more private browsing experience. It also includes frequent updates and security patches, maintaining compatibility with Chrome extensions. With a lightweight footprint and advanced configuration options, Thorium is ideal for users who want a streamlined, customizable, and efficient browsing tool without sacrificing functionality. It combines Chromium’s power with refined performance and stronger user privacy by design.</p>
+  </description>
+  <developer id="rocks.thorium">
+    <name>Thorium Project</name>
+    <url type="homepage">https://thorium.rocks/</url>
+  </developer>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-3-Clause</project_license>
+  <url type="homepage">https://thorium.rocks/</url>
+  <icon type="stock">thorium</icon>
+  <categories>
+    <category>Network</category>
+    <category>WebBrowser</category>
+  </categories>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+    <content_attribute id="money-advertising">none</content_attribute>
+  </content_rating>
```

Comment:
```suggestion
  <content_rating type="oars-1.1" />
```

Reply:
on that note, doesn't this come with chromium's telemetry or is that patched out?

Reply:
It does have some telemetry, but where would that be set? Don't I need to specify content_attribute for all of these? Unclear why the suggestion is to delete it.